cmd/telegram.go

```
@@ -0,0 +1,51 @@
+	"github.com/abhinavxd/libredesk/internal/inbox/channel/telegram"
+	"github.com/valyala/fasthttp"
+	"github.com/zerodha/fastglue"
+)
+
+func handleTelegramWebhook(r *fastglue.Request) error {
+	var app = r.Context.(*App)
+
+	inboxID, err := strconv.Atoi(r.RequestCtx.UserValue("id").(string))
+	if err != nil || inboxID == 0 {
+		r.RequestCtx.SetStatusCode(fasthttp.StatusBadRequest)
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 16) but it returns nil

(nilerr)